spec/controllers/github_contributors_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+include LogInHelper
+
+describe GithubContributorsController do
+  before(:each) do
+    student.save
+    instructor.save
+    @role = Role.new(name: 'Instructor')
+    @role.save
+    @instructor = User.find_by(name: 'instructor')
+    @student = User.find_by(name: 'student')
+    @assignment = Assignment.where(name: 'My assignment').first ||
+        Assignment.new("name" => "My assignment", "instructor_id" => @instructor.id)
+    @assignment.save
+    @assignment_team = AssignmentTeam.where(name: 'My Github Team').first ||
+        AssignmentTeam.new(name: 'My Github Team', parent_id: @assignment.id)
+    @assignment_team.save
+    @sub_record = SubmissionRecord.new(content: 'https://github.com/mozilla/geckodriver',
+                                       operation: 'Submit Hyperlink',
+                                       team_id: @assignment_team.id,
+                                       user: @student.name,
+                                       assignment_id: @assignment.id)
+    @sub_record.save
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@instructor)
```

Comment:
Actually, these 3 lines have been implemented in `stub_current_user` in [rails_helper.rb](https://github.com/expertiza/expertiza/blob/master/spec/rails_helper.rb#L69)